operator/src/main/java/org/bf2/operator/operands/KafkaCluster.java

```diff
@@ -617,7 +618,13 @@ private void addQuotaConfig(ManagedKafka managedKafka, Kafka current, Map<String
 
         // Check storage every storageCheckInterval seconds
         KafkaInstanceConfiguration instanceConfig = this.configs.getConfig(managedKafka);
-        config.put("client.quota.callback.static.storage.check-interval", String.valueOf(instanceConfig.getStorage().getCheckInterval()));
+        if (instanceConfig.getKafka().isEnableQuota()) {
+            KafkaInstanceConfiguration.Kafka kafka = instanceConfig.getKafka();
+            config.put("client.quota.callback.usageMetrics.topic", String.valueOf(kafka.getQuotaCallbackUsageMetricsTopic()));
+            config.put("client.quota.callback.quotaPolicy.check-interval", String.valueOf(kafka.getQuotaCallbackQuotaPolicyCheckInterval()));
+            config.put("client.quota.callback.kafka.clientIdPrefix", String.valueOf(kafka.getQuotaCallbackQuotaKafkaClientIdPrefix()));
+            config.put("client.quota.callback.static.storage.check-interval", String.valueOf(instanceConfig.getStorage().getCheckInterval()));
```

Comment:
Just to mention it, the `OperandOverride` class supports `additionalProperties` that can be used to configure properties specific to a Strimzi bundle deployment version. 

That can be leveraged to only apply new/changed properties alongside the image updates they align with. I think ultimately we'll want to use that (or something like it) to extend support for blast-radius control to include configuration as well.